nc-games/src/components/AllReviews/AllReviews.jsx

```diff
@@ -10,17 +10,25 @@ export const AllReviews = () => {
   const [reviews, setReviews] = useState([]);
   const [theCategory, setTheCategory] = useState(undefined);
   const [loading, setLoading] = useState(true);
+  const [isError, setIsError] = useState(false);
 
   useEffect(() => {
     setLoading(true);
-    getReviews(theCategory).then((ApiReviews) => {
-      setLoading(false);
-      setReviews(ApiReviews.reviews);
-    });
+    getReviews(theCategory)
+      .then((ApiReviews) => {
+        setIsError(true);
+        setLoading(false);
```

Comment:
probably best to setLoading after setReviews, but it wouldn't make much of a difference really